### ISSUE TITLE
Blockly implementation

### ISSUE THREAD

**Issue by blacksheepinc** [mycontroller-org/mycontroller#388]
As discussed, please inmplement blockly as a more user friendly way to create rules.
Source: http://forum.mycontroller.org/topic/159/integrate-blockly/11
Thanks

**Comment by blacksheepinc** [comment]
Any news about this?

**Comment by jkandasa** [comment]
@blacksheepinc No, Sorry for the late response. Till now I do not find great time complete this feature.

**Comment by seant100** [comment]
I am also interested in this feature being added. Any idea if it will be at some stage?
